web/src/components/graph/areagraph.jsx

```diff
@@ -186,7 +190,7 @@ const AreaGraph = React.memo(
           transform={`translate(-1 ${containerHeight - 1})`}
           className="x axis"
           selectedTimeAggregate={selectedTimeAggregate}
-          datetimes={datetimes}
+          datetimes={datetimesWithNext}
```

Comment:
the time axis is passed the "augmented" datetimes array so it renders the last point (i.e. if the last datapoint is 2021, we still need to render the label "2022" so we can show data from 2021 to 2022)